deepchem/utils/openvino_model.py

```diff
@@ -200,17 +252,42 @@ def __call__(self, generator: Iterable[Tuple[Any, Any, Any]]):
         inputs, _, _ = self._keras_model._prepare_batch((inputs, None, None))
       elif self._torch_model is not None:
         inputs, _, _ = self._torch_model._prepare_batch((inputs, None, None))
-      inputs = inputs[0]
-
-      # Last batch size may be less or equal than overall batch size.
-      # Pad extra values by zeros and cut at the end.
-      last_batch_size = inputs.shape[0]
-      if last_batch_size != self._batch_size:
-        assert (last_batch_size < self._batch_size)
-        inp = np.zeros(
-            [self._batch_size] + list(inputs.shape[1:]), dtype=np.float32)
-        inp[:last_batch_size] = inputs
-        inputs = inp
+
+      inp_dict = {}
+      if (type(self._torch_model).__name__ == 'CGCNNModel'):
+        self._torch_model.model.graph = inputs
+
+        node_feats = inputs.ndata.pop('x')
+        edge_feats = inputs.edata.pop('edge_attr')
+
+        inputs = [node_feats, edge_feats]
+      else:
+        assert (len(self._exec_net.input_info) == 1), 'Not implemented'
+        inputs = [inputs[0]]
+
+      if len(self.input_size) == 0:
```

Comment:
Seems always true at this moment, as for me

Reply:
Why? It's true only for first iteration

Reply:
Maybe better to compare with `self.net` input info?